### PR TITLE
Added npm prep

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+  firefox: "52.0.2"
+
 branches:
   only:
     - master
     - /^[0-9]+\.[0-9]+\.[0-9]+.*/
+
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
```
Comment (Blackbaud-SteveBrush) [inline .travis.yml]: @Blackbaud-BobbyEarl Do I still need this bit, for headless browser support?

Comment (Blackbaud-BobbyEarl) [reply .travis.yml]: Safe to remove these lines.

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,20 @@
 language: node_js
 
+addons:
+  firefox: "52.0.2"
```
Comment (Blackbaud-BobbyEarl) [inline .travis.yml]: You can also remove this.

File: scripts/prepare-package.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+const rootPath = path.join(__dirname, '..');
+const distPath = path.join(rootPath, 'dist');
+
+function createDist() {
+  if (!fs.existsSync('dist')) {
+    fs.mkdirSync('dist');
+  }
+
+  fs.copySync(path.join(rootPath, 'src/code-block.js'), path.join(distPath, 'index.js'));
```
Comment (Blackbaud-BobbyEarl) [inline scripts/prepare-package.js]: I would change this to `path.join(rootPath, 'src', 'code-block.js')`.

File: .travis.yml (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply .travis.yml]: Done.

File: scripts/prepare-package.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+const fs = require('fs-extra');
+const path = require('path');
+
+const rootPath = path.join(__dirname, '..');
+const distPath = path.join(rootPath, 'dist');
+
+function createDist() {
+  if (!fs.existsSync('dist')) {
+    fs.mkdirSync('dist');
```
Comment (Blackbaud-BobbyEarl) [inline scripts/prepare-package.js]: I would change `'dist'` to use `distPath`.

**Review by Blackbaud-BobbyEarl — CHANGES_REQUESTED** [review]

**Comment by Blackbaud-SteveBrush** [comment]
@Blackbaud-BobbyEarl Ready for another look, thanks!

**Review by Blackbaud-BobbyEarl — APPROVED** [review]

**Pull request merged**